go/vt/vttablet/tabletmanager/vreplication/vplayer_flaky_test.go

```diff
@@ -2505,28 +2505,9 @@ func TestTimestamp(t *testing.T) {
 	expectData(t, "t1", [][]string{{"1", want, want}})
 }
 
-func shouldRunJSONTests(t *testing.T, name string) bool {
-	skipTest := true
-	flavors := []string{"mysql80", "mysql57"}
-	for _, flavor := range flavors {
-		if strings.EqualFold(env.Flavor, flavor) {
```

Comment:
`env.Flavor` contains the GTID flavor which is always `MySQL56` and never any of the values we check against. So we were never running these tests.